.azure-pipelines/cd-build-deploy-typescript.yml

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Build, package, and deploy TypeScript packages for @microsoft/agents-m365copilot-core and @microsoft/agents-m365copilot-beta
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - 'typescript/packages/agents-m365copilot-beta/generated/**'
+      - 'typescript/packages/agents-m365copilot-core/src/**'
+  tags:
+    include:
+      - '*microsoft/agents-m365copilot-beta'
```

Comment:
same comment about duplicated triggers

Reply:
That's okay, I've added conditions to the tasks to limit which tasks are run based on the tags added by RP. Probably would be better to make separate pipelines for each library, will address that later.

Reply:
your condition will match both tags